app/feeds/new/license/template.hbs

```diff
@@ -10,6 +10,11 @@
 		<p>
 			{{documentation-link "add-feed.md#identify-the-license-for-the-feed" "Learn more about licenses"}}
 		</p>
+		{{#if feedExists}}
+			<div class="feed-exists-warning">
+				It looks like this feed already exists in the datastore! Please add to or edit the information we currently have.
```

Comment:
@drewda I wasn't sure if we wanted to customize the copy in each place. If we're going to use the same copy, I can create a component.